### ISSUE TITLE
Initial setup should inspire more confidence

### ISSUE THREAD

**Issue by luisivan** [aragon/issues#47]
<img width="1305" alt="aragon_and_aragon" src="https://cloud.githubusercontent.com/assets/718208/25988764/92902b5c-36f9-11e7-8e01-da3d6c87751f.png">

_Picture source: `skwp` from our Slack_

**Comment by sinanku** [comment]
I agree, first impressions are important.  Perhaps taking inspiration from the Slack on-boarding UI?
